CHANGELOG.md

```diff
@@ -1,5 +1,29 @@
 ## 4.63.0 (Unreleased)
 
+NOTES:
+* alloydb: changed `location` from `optional` to `required` for `google_alloydb_cluster`. `location` had previously been marked as optional, but operations failed if it was omitted, and there was no way for `location` to be inherited from the provider configuration or from an environment variable. This means there was no way to have a working configuration without `location` specified. ([#14330](https://github.com/hashicorp/terraform-provider-google/pull/14330))
+
+FEATURES:
+* **New Resource:** `google_access_context_manager_ingress_policy` ([#14302](https://github.com/hashicorp/terraform-provider-google/pull/14302))
+* **New Resource:** `google_compute_public_advertised_prefix` ([#14303](https://github.com/hashicorp/terraform-provider-google/pull/14303))
+* **New Resource:** `google_compute_public_delegated_prefix` ([#14303](https://github.com/hashicorp/terraform-provider-google/pull/14303))
+* **New Resource:** `google_compute_region_commitment` ([#14301](https://github.com/hashicorp/terraform-provider-google/pull/14301))
+* **New Resource:** `google_network_services_http_route` ([#14294](https://github.com/hashicorp/terraform-provider-google/pull/14294))
+* **New Resource:** `google_networkservices_tcp_route` (beta) ([#14342](https://github.com/hashicorp/terraform-provider-google/pull/14342))
```

Comment:
Beta-only should be removed

Reply:
Also the product name doesn't match the line above it

Reply:
Thanks for spotting this!!

Reply:
Addressed in [07060f6](https://github.com/hashicorp/terraform-provider-google/pull/14382/commits/07060f6ac816fa4c01ffc556467830f72f6929d4)